bucket/nohboard.json

```
@@ -0,0 +1,29 @@
+{
+    "version": "1.3.0",
+    "description": "A keyboard visualizer",
+    "homepage": "https://github.com/ThoNohT/NohBoard",
+    "license": "GPL-2.0-only",
+    "url": "https://github.com/ThoNohT/NohBoard/releases/download/v1.3.0/NohBoard-ReWrite-v1.3.0.zip",
+    "hash": "d868ada36466e3f88d3edb023d333fac503fa48f282c27dae6148d044a65b59d",
+    "pre_install": [
+        "# Create NohBoard.json if it does not exist",
+        "if (!(Test-Path \"$persist_dir\\NohBoard.json\" -PathType Leaf)) {",
```

```suggestion
        "if (!(Test-Path \"$persist_dir\\NohBoard.json\")) {",
```
-PathType Leaf looks redundant. I cannot think of any case it can be a folder/symlink.